### PR TITLE
Update logo page

### PR DESCRIPTION
**Pull request by Webysther**
- New images in SVG and PNG. Expected Colin override with new one soon.
- Update page logos with licensing information.
- Removed old logos and icons from page, maintain on server (images/logos) for compatibility.
- Add link of image in Wikimedia Commons to download as fallback bandwidth

### CODE REVIEW

**Comment by Webysther** [comment]
@bjori @morrisonlevi

**Comment by morrisonlevi** [comment]
We already have the logo in SVG format inside the repository; see http://php.net/images/logos/php-logo.svg.

**Comment by morrisonlevi** [comment]
On second look it is a different logo. The SVG included here is the logo with the purple circle around it. It still has Inkscape attribtutes in it, which seems odd.

File: download-logos.php [hunk 1]
```diff
@@ -66,290 +40,42 @@ function random_bgcolor($min, $max)
...
-    <li>
-     48x48, 32x32, 24x24, 16x16 sizes, in each of 16 colour, 256 colour,
-     and true colour formats... all in one .ICO file
-    </li>
-    <li>these icons are made by Jonathan Metillon</li>
-    <li><a href="/images/logos/php_xpstyle.ico">XP ICO format</a></li>
-   </ul>
+   <img src="/images/logos/new-php-logo.png" width="200" alt="php logo">
+   <br>
+   <a href="https://upload.wikimedia.org/wikipedia/commons/2/27/PHP-logo.svg">SVG format</a> |
```
Comment (bjori) [inline download-logos.php]: I don't think its kosher to link to external sources for our canonical graphics

**Comment by Webysther** [comment]
@morrisonlevi I used this one [here](https://commons.wikimedia.org/wiki/File:PHP-logo.svg) on Wikimedia.
I hope Colin override in the future this version.

You have a better version? Colin said he have, but don't send.

File: download-logos.php (hunk 1 above)
Comment (Webysther) [inline download-logos.php]: Is just to save network bandwidth, the Wikimedia have more redundant servers, 
but I can change the point to the pictures I added. No problem.

**Comment by Webysther** [comment]
This one http://php.net/images/logos/php-logo.svg is made by Colin? 
This version maybe show as alternative one, looks better to print and with another background.

File: download-logos.php (hunk 1 above)
Comment (Webysther) [inline download-logos.php]: Fixed.

**Comment by Webysther** [comment]
@morrisonlevi Exist better bold version?

captura de tela de 2016-06-02 17-09-50

**Comment by Webysther** [comment]
Now look great in mobile.

screencapture-localhost-8080-download-logos-php-1464898308045

**Comment by Webysther** [comment]
Change to work using grid and minor fixes in text.

screencapture-localhost-8080-download-logos-php-1464901253965
screencapture-localhost-8080-download-logos-php-1464901194573

**Comment by salathe** [comment]
> Removed old logos and icons from page

Why?

**Comment by salathe** [comment]
Can you remove the whitespace changes, no need for those to be mixed in with the patch?

**Comment by Webysther** [comment]
1 - Because SVG format anyone can generate any size and other format.
2 - Fixed

**Comment by Webysther** [comment]
@morrisonlevi you can check if is all ok?

**Comment by bjori** [comment]
Did anyone ever confirm anything about that second "logo" ?

Its definitely not the php logo and I don't think its should be on this page at all, and its not made by Colin so I don't think his announcement applies to it?

**Comment by Webysther** [comment]
The second logo is used on website, is a derivative work from Colin, i put on this page to make possible to other people to download the logo of website.

@bjori the @morrisonlevi is author of this version:

capturar

As the image that @morrisonlevi is a devirado work, you might want to put that he is the author and let explicitly that it is a derivative work that is being used on the site.

An alternative is to place the original image, but I do not know if it would be good.

All right place as derived image for use on the website?

**Comment by Webysther** [comment]
@morrisonlevi @bjori @salathe any news?

**Comment by salathe** [comment]
The SVG logo added in this PR ("new-php-logo.svg") is not the one that Colin provided [1] in the mailing list thread.

Also, Colin released the logo as CC BY-SA 4.0, but the logo provided by him in the mailing list references CC BY-SA 3.0 inside the SVG file. The logo currently in this PR doesn't embed any license information, which would be good to have.

Finally, as an aside, I would prefer to see the old "powered by" and "icon" style images kept available and referenced on the logos page somewhere, for posterity.

[1] http://markmail.org/message/x4gda3kakjwuqi2u

**Comment by amin007** [comment]
new logo? good

**Comment by EliW** [comment]
Poking @Webysther @salathe and @bjori to get this request moving forward again. :)

**Comment by salathe** [comment]
FWIW, I'm still waiting on a response from my previous message from Jan 27th.

**Comment by Webysther** [comment]
@salathe you response:

- The difference about CC 3 and 4 not relevant, but, i put the original version, this version is the same i made optimizations e send to [Wikimedia Commons](https://commons.wikimedia.org/wiki/File:PHP-logo.svg).

- Fixed text about the license version inside file is sufficient?

- I made the change to keep page with older images and with disclaimer: is not recommended to use because are obsolete.

**Comment by salathe** [comment]
> @salathe, I had not understood that it was a requirement to carry out...

It was not, but I was expecting a reply of some kind. 

> I made the change to keep page with older images and with disclaimer: is not recommended to use because are obsolete.

I don't see this change, did you forget to push here?

**Comment by Webysther** [comment]
@salathe I push all this changes in few hours.

**Comment by Webysther** [comment]
@salathe Done.

screencapture-localhost-8080-download-logos-php-1489629313012
screencapture-localhost-8080-download-logos-php-1489629282760

**Comment by amin007** [comment]
where i can find logo php in cdn?

**Comment by Webysther** [comment]
@amin007 yes, [Wikimedia commons](https://commons.wikimedia.org/wiki/File:PHP-logo.svg) isn't a CDN, but maybe works for you.

**Comment by Webysther** [comment]
@morrisonlevi @bjori @salathe any news?

**Comment by Webysther** [comment]
Ping @morrisonlevi @bjori @salathe

**Comment by petk** [comment]
Thanks for this pull requests as slow as they seem to get into production. SVG format is definitely missing there. Each time I need a PHP logo to use somewhere, I now use Wikimedia and other sites instead which is kind of strange that PHP don't have such important thing on the main page. Hope this gets on php.net in some form or the other. Cheers.

**Pull request merged**